--- a/Makefile
+++ b/Makefile
@@ -34,11 +34,11 @@
 install-headers:
 	mkdir -p $(DESTDIR)$(PREFIX)/include/SDL
 	cp -rp include/*.h $(DESTDIR)$(PREFIX)/include/SDL
+	cp -rp sdl.pc $(DESTDIR)$(PREFIX)/lib/pkgconfig/sdl.pc
+	cp -rp sdl-config $(DESTDIR)$(PREFIX)/bin/sdl-config
 
 install-lib:
 	cp -rp $(TARGET) $(DESTDIR)$(PREFIX)/lib/
-	cp -rp sdl.pc $(DESTDIR)$(PREFIX)/lib/pkgconfig/sdl.pc
-	cp -rp sdl-config $(DESTDIR)$(PREFIX)/bin/sdl-config
 
 clean:
 	rm -f $(TARGET) *.o *.a *.so
